x-pack/legacy/plugins/canvas/external_runtime/components/footer/page_preview.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useExternalEmbedState } from '../../context';
+import { Page } from '../page';
+import { setPage } from '../../context/actions';
+import { CanvasPage } from '../../types';
+
+// @ts-ignore CSS Module
+import css from './page_preview.module';
+
+interface Props {
+  number: number;
+  height: number;
+  page: CanvasPage;
+}
+
+export const PagePreview = ({ number, page, height }: Props) => {
+  const [{ workpad }, dispatch] = useExternalEmbedState();
+  if (!workpad) {
+    return null;
+  }
+
+  const { height: workpadHeight, width: workpadWidth } = workpad;
+  const scale = height / workpadHeight;
```

Comment:
Will address.

Reply:
I remember now why I did this: because the height is constant: `100px`.  So I only have to scale the thumbnail to that height.  The `Math.max` is actually unnecessary.